### ISSUE TITLE
Error: looks like "https://svc-catalog-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: failed to fetch https://svc-catalog-charts.storage.googleapis.com/index.yaml : 403 Forbidden

### ISSUE THREAD

**Issue by fanchenxu-ship** [kubernetes-sigs/service-catalog#2869]
# Bug Report 

**What happened**:
 ` when i run "helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com"` it failed.
**Error message**:
` Error: looks like "https://svc-catalog-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: failed to fetch https://svc-catalog-charts.storage.googleapis.com/index.yaml : 403 Forbidden`
**Environment**:
- helm version: v3.4.2

**Comment by jiayihu** [comment]
What worked from me was downloading the chart from https://github.com/kubernetes-sigs/service-catalog/tree/master/charts/catalog and installing it locally. 
I also changed the image to `image: quay.io/kubernetes-service-catalog/service-catalog-arm64:v0.3.1` since i'm using ARM64/aarch64

**Comment by achrinza** [comment]
This should be updated command:

```
helm repo add service-catalog https://kubernetes-sigs.github.io/service-catalog
```

**Comment by kjaksik** [comment]
It also happens during CI pipelines. Helm in version 2.16.7 had the repo hard coded and `helm init` tried to reach this repo. Should be fixed in #2873

**Comment by jhvhs** [comment]
Fixed the documentation in #2875
